python/ray/serve/scripts.py

```diff
@@ -176,6 +179,87 @@ def deploy(config_file_name: str, address: str):
         log_failed_request(response, address)
 
 
+@cli.command(
+    help="[Experimental] Run deployments via Serve's Python API.",
+    hidden=True,
+)
+@click.option(
+    "--config_file_path",
+    "-c",
+    default=None,
+    required=False,
+    type=str,
+    help="Path to a Serve YAML configuration file.",
+)
+@click.option(
+    "--import_path",
+    "-i",
+    default=None,
+    required=False,
+    type=str,
+    help=(
+        "Import path to class or function to deploy. Must be of form "
+        '"module.submodule_1...submodule_n.MyClassOrFunction".'
+    ),
+)
```

Comment:
can we make this a positional instead? so you always do: `serve run <config_or_import_path>`

Reply:
Sure, I switched it to a positional argument.